pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,113 @@
 
 ```yml
-- command: "command.sh"
+steps:
+  - command: "command.sh"
 ```
 
-The following step includes every configuration property for *command* steps:
+
+## Command Step Attributes
+
+_Optional attributes:_
+
+<table>
+  <tr>
+    <td><code>name</code></td>
+    <td>
+      The text and/or emoji that will be displayed in the pipeline visualisation in Buildkite.<br>
+      <em>Example:</em> <code>"\:hammer\: Tests"</code><br>
+    </td>
+  </tr>
+  <tr>
+    <td><code>branches</code></td>
+    <td>
+      The <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch pattern</a> defining which branches will include this command step in their builds.<br>
+      <em>Example:</em> <code>"master stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>env</code></td>
+    <td>
+      A list of <a href="/docs/builds/environment-variables">environment variables</a> for this step. <br>
+      <em>Example:</em> <code>FOO: "bar"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>agents</code></td>
+    <td>
+      A list of <a href="/docs/agent/cli-meta-data">meta-data</a> key value pairs to <a href="docs/agent/cli-start#agent-targeting">target specific agents</a> for this step. <br>
+      <em>Example:</em> <code>npm: "true"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>plugins</code></td>
+    <td>
+      A list of plugins that you require for this step. <br>
+      <em>Example:</em> <code>docker-compose</code><br>
+      <em>Example:</em> <code>harrietgl/myplugin</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>artifact_paths</code></td>
+    <td>
+      The path or paths where <a href="/docs/agent/cli-artifact">artifacts</a> from this step will be uploaded. <br>
+      <em>Example:</em> <code>"logs/**/*;coverage/**/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>parallelism</code></td>
+    <td>
+      The number of jobs that will run this step <a href="/docs/builds/parallelizing-builds#job-parallelism">simultaneously</a>. <br>
+      <em>Example:</em> <code>1</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>concurrency</code></td>
+    <td>
+      The <a href="/docs/builds/controlling-concurrency#concurrency-limits">maximum number of jobs</a> that can be run inside this concurrency group at the same time as this job. <br>
+      <em>Example:</em> <code>1</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>concurrency_group</code></td>
+    <td>
+      A nickname for the concurrency value that you set with the <code>concurrency</code> attribute. This can also be the name of an existing <a href="/docs/builds/controlling-concurrency#concurrency-groups">concurrency group</a>. <br>
+      <em>Example:</em> <code>"serial_tests"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>timeout_in_minutes</code></td>
+    <td>
+      The number of minutes this step should wait for an agent before timing out. When the step hits this timeout without finding an agent, the step will fail. <br>
```

Comment:
Considering that we don't have a 'cancelled' state, I'd like to avoid using that word... but I can't come up with anything better haha. Maybe something like 

If the job does not finish within this time limit
it will automatically fail
it will fail, therefore failing the build
it will fail, which causes the build to fail
the job will be marked as failed

Or something